frontend/playwright-report/index.html

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
Not sure that anything in the /playwright-report folder should actually be checked into git and therefor be part of the PR.